Frontmatter: use numbered affiliations

dhimmel opened the pull request:
Switch frontmatter to condensed format which uses numbered affiliations.

Refs https://github.com/greenelab/deep-review/issues/700

dhimmel commented:
I tried a version that hyperlinked the ORCID and GitHub icons for each author using the following jinja2/md:

```
{%- if author.orcid is defined -%}
[![ORCID icon](images/orcid.svg){height="10px" width="10px"}](https://orcid.org/{{author.orcid}})
{%- endif -%}
{%- if author.github is defined -%}
[![GitHub icon](images/github.svg){height="10px" width="10px"}](https://github.com/{{author.github}})
{%- endif -%}
{%- if author.twitter is defined -%}
[![Twitter icon](images/twitter.svg){height="10px" width="10px"}](https://twitter.com/{{author.twitter}})
{%- endif -%}
```

I didn't love the look:

![screenshot from 2018-01-11 15-11-21](https://user-images.githubusercontent.com/1117703/34845772-0bc4508e-f6e4-11e7-830a-90a0c75bcb46.png)

What do others think.

I also considered hyperlinking authors to ORCIDs, but then the names are blue.

dhimmel commented:
@agitter I was thinking we'd add additional symbols as superscripts following the affiliation numbers. Who should have what superscripts?

agitter commented:
Maybe the icons would look less crowded if you include ORCID but not GitHub?  That's what [bioRxiv does](https://doi.org/10.1101/142760) and it looks nice there.  They also have the icon precede the author name, which I'm indifferent about.

The uploaded manuscript was a manubot-rootstock build.

I think we can use the same superscripts as last time unless anyone has other suggestions:
- Travers Ching <sup>*</sup>
- Anthony Gitter <sup>†</sup>
-  Casey S. Greene <sup>†</sup>

<sup>*</sup> Author order was determined with a randomized algorithm

<sup>†</sup> To whom correspondence should be addressed: gitter@biostat.wisc.edu (AG) and csgreene@upenn.edu (CSG)

dhimmel commented:
I added ORCID icons before author names in https://github.com/greenelab/deep-review/pull/769/commits/fa6432e9d77e1f16b1a6f3298e30532d721388bf. Unfortunately, I couldn't get nonbreaking spaces to work so some author names are on different lines than their ORCID icon.

I added the symbols in https://github.com/greenelab/deep-review/pull/769/commits/8484b3472a8b5ec04aee659ace3e9ac430611aaf. Note that I had to remove the `@` symbols due to https://github.com/greenelab/manubot/issues/25. I also switched the authorship symbol to something slightly more traditional: ☯ as opposed to *. Can change back if others do not like.

Here's the latest look:

![authors](https://user-images.githubusercontent.com/1117703/34848661-8dfa9e92-f6ed-11e7-8a99-9a460a537ad5.png)

agitter commented on content/00.front-matter.md:
```diff
@@ -21,9 +21,12 @@ on {{date}}.
 {%- if not loop.last -%}, {%- endif -%}
 {% endfor %}
+
+<sup>☯</sup> --- Author order was determined with a randomized algorithm<br>
```
We can change ☯ back to * later if someone objects.  The way you implemented the superscripts is easy to change.

agitter commented on content/00.front-matter.md:
```diff
@@ -21,9 +21,12 @@ on {{date}}.
 {% endfor %}
+
+<sup>☯</sup> --- Author order was determined with a randomized algorithm<br>
+<sup>†</sup> --- To whom correspondence should be addressed: `gitter` at `biostat.wisc.edu` (AG) and `csgreene` at `upenn.edu` (CSG)
```
This workaround looks good to me.  We could use `gitter at biostat.wisc.edu` instead of `gitter` at `biostat.wisc.edu`, but I don't have a strong preference.
